scripts/globals/abilities/super_jump.lua

```diff
@@ -22,4 +23,18 @@ end;
 -----------------------------------
 
 function onUseAbility(player,target,ability)
-end;
\ No newline at end of file
+
+    -- Reduce 99% of total accumulated enmity
+    if (target:isMob()) then
+        target:lowerEnmity(player, 99);
+    end
+    
+    ability:setMsg(0);
+    
+    -- If the Dragoon's wyvern is out and alive, tell it to use Super Climb
+    local wyvern = player:getPet();
+    if (wyvern ~= nil and player:getPetID() == PET_WYVERN and wyvern:getHP() > 0) then 
+        wyvern:useJobAbility(636, wyvern);
+    end
+
```

Comment:
To add to above: anything that was mid cast on a drg that just popped super jump will cancel, and any targeting that had them focused is lost.